src/main/resources/changelog/changes.v3.0.0/add-optional-filed-to-dependency-table.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+  <changeSet id="add-optional-filed-to-dependency-table" author="okuzminov">
+    <addColumn tableName="application_dependency">
+      <column name="optional" type="boolean"/>
+    </addColumn>
+  </changeSet>
+
+  <changeSet id="populate-optional-for-dependency" author="okuzminov">
```

Comment:
Hi @OleksiiKuzminov , to set up default value it is possible to use defaultValueBoolean="false" for addColumn definition